Source/WebKit/UIProcess/ios/forms/WKDateTimeInputControl.mm

```diff
@@ -341,11 +341,11 @@ - (instancetype)initWithView:(WKContentView *)view
         mode = (UIDatePickerMode)UIDatePickerModeYearAndMonth;
         break;
     default:
-        [self release];
         return nil;
```

Comment:
Why is this change OK?

Reply:
Because it is in an init method, this would be an overrelease.

Reply:
Please elaborate. What else releases the object?

Reply:
It is never initialized in the first place is what I mean, so releasing it makes no sense.

Reply:
Actually I was wrong. This should stay. @aproskuryakov